test/org/loklak/tools/storage/UTF8Test.java

```diff
@@ -0,0 +1,52 @@
+package org.loklak.tools.storage;
```

Comment:
@sakshee-19 This should be ```org.loklak.tools```, since the original file does not reside in ```storage``` but in ```tools```.

Reply:
Also the file should be outside ```storage```.